storage/connectors/jdbc/pom.xml

```diff
@@ -146,6 +162,12 @@
             <artifactId>spring-test</artifactId>
             <scope>test</scope>
         </dependency>
+<!--        For testing DB only -->
+        <dependency>
+            <groupId>org.xerial</groupId>
+            <artifactId>sqlite-jdbc</artifactId>
+            <version>3.16.1</version>
+        </dependency>
     </dependencies>
     <properties>
         <maven.test.skip>true</maven.test.skip>
```

Comment:
I think we should un-skip the new mocking tests. 
And skip those manual tests, assume we still want to include those manual e2e tests in our snowflake branch.